spec/openapi.yaml

```diff
@@ -1759,9 +1761,21 @@ paths:
                           ip:
                             type: string
                             description: Server's IP address
+                          staticIp:
+                            type: boolean
+                            enum:
+                              - true
+                              - false
+                            description: If ``treu`` node have a static IP address
```

Comment:
Check typo in description. 

`description: If ``true`` node has a static IP address`